helm/mds/templates/deployment.yaml

```diff
@@ -0,0 +1,102 @@
+{{- range $api := .Values.apis }}
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ $api.name }}
+  labels:
+    app: {{ $api.name }}
+  namespace: {{ $.Release.Namespace }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: {{ $api.name }}
+  template:
+    metadata:
+      labels:
+        app: {{ $api.name }}
+    spec:
+      containers:
+      - name: user-container
```

Comment:
The container name recently changed on sandbox. It used to be `user-container`. Is this PR missing some changes that are currently deployed? This is current from Kibana.
<img width="223" alt="Screen Shot 2019-09-24 at 12 37 46 PM" src="https://user-images.githubusercontent.com/3439869/65531583-3bd2b180-dec8-11e9-96f4-87fa4ba92116.png">

Reply:
Updated to account for this